apps/src/templates/feedback/Code.jsx

```diff
@@ -0,0 +1,17 @@
+var msg = require('../../locale');
+
+var Code = React.createClass({
+  
+  propTypes: {
+    message: React.PropTypes.string.isRequired,
+    code: React.PropTypes.string.isRequired,
+  },
+
+  render: function () {
+    return (<div className="generated-code-container">
+      <p className="generatedCodeMessage" dangerouslySetInnerHTML={{__html: this.props.message}} />
+      <pre className="generatedCode" dangerouslySetInnerHTML={{ __html: this.props.code }} />
+    </div>);
```

Comment:
I'm wondering at what point we should start worrying about how frequently we're using dangerouslySetInnerHTML. It's starting to sound a lot less dangerous to me than it once did as it becomes common place. What do you think?